cmd/tfimport/main.go

```diff
@@ -16,7 +16,7 @@
 // Requires Terraform to be installed and for authentication to be configured for each provider in the Terraform configs provider blocks.
 //
 // Usage:
-// $ go run . [--input_dir=/path/to/config]
+// $ go run . [--input_dir=/path/to/config] [--resource_types 'google_storage_bucket.bucket' --resources 'google_resource_manager_lien.lien']
```

Comment:
Have you tested that listing the resource multiple times works as intended, or do we need to list it once and comma separate?

Reply:
I have tested, it does work as intended when specified multiple times.